lib/cocoapods/bazel/target.rb

```diff
@@ -594,6 +596,24 @@ def app_targeted_device_families
         end
       end
 
+      def vendored_xcframeworks
+        pod_target.xcframeworks.values.flatten(1).uniq.map do |xcframework|
+          {
+            'name' => xcframework.name,
+            'slices' => xcframework.slices.map do |slice|
+              {
+                'identifier' => slice.identifier,
+                'platform' => slice.platform.name.to_s,
+                'platform_variant' => slice.platform_variant.to_s,
+                'supported_archs' => slice.supported_archs,
+                'path' => slice.path.relative_path_from(@package_dir).to_s,
+                'build_type' => { 'linkage' => slice.build_type.linkage.to_s, 'packaging' => slice.build_type.packaging.to_s }
```

Comment:
`Pod::Xcode::XCFramework::Slice` also has a `headers` attr, do we care about using/mapping that here?

Reply:
I don't think we need to, it'll get picked up by the `**/*` glob added in the macro -- similar to what happens for other vendored frameworks